src/pie-chart/__integ__/with-links.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+
+function setupTest(useLinks: string, testFn: (page: BasePageObject) => Promise<void>) {
+  return useBrowser(async browser => {
+    const page = new BasePageObject(browser);
+    await browser.url(`#/light/pie-chart/with-links?useLinks=${useLinks}`);
+    await testFn(page);
+  });
+}
+
+describe('Pie chart with links in key-value pairs', () => {
+  describe.each(['keys', 'values'])('with links in the %s', useLinks => {
+    it(
+      'allows to focus internal popover content',
+      setupTest(useLinks, async page => {
+        const expectedFocusedText =
+          useLinks === 'keys'
+            ? // The external link icon adds an extra space at the end
+              'Popularity '
+            : '50% ';
+        await page.click('#focus-target');
+        await page.keys(['Tab', 'Enter']);
```

Comment:
I would add code comments to explain what a sequence of keys is for, e.g.
```
// Pin popover on the first chart segment
await page.keys(['Tab', 'Enter']);
// Move focus to the second interactive element of the popover
await page.keys('Tab');
```

Reply:
Done